app/scripts/modules/core/src/insight/insight.less

```diff
@@ -17,7 +17,9 @@ insight-layout {
 }
 
 ui-view[name="insight"],
-react-ui-view-adapter[name="insight"] {
+react-ui-view-adapter[name="insight"],
```

Comment:
This essentially adds the same as the "flex-fill" class to these specific ui-view children